docs/install/build_from_source.md

```diff
@@ -28,184 +32,102 @@ You need C++ build tools and BLAS library to build MXNet shared library. If you
    for Windows) to build the library.
```

Comment:
A C++ compiler that supports C++ 11.
[G++ (4.8 or later)](https://gcc.gnu.org/gcc-4.8/), [Clang](http://clang.llvm.org/) or [Visual Studio 2015](https://visualstudio.microsoft.com/downloads/) (on Windows) is required.

Reply:
Can we also make CMake the default (instead of GNU Make and cmake only for Windows)?

Reply:
I'd be happy to do that, but so many other downstream instructions use `make`. Are we ready to update all of those too?

Reply:
Well, maybe it can wait then and be done in a separate PR